src/initialState.js

```diff
@@ -4,12 +4,38 @@ const initialState = {
   projects: [
     {
       id: uuid(),
-      name: 'Sample Project',
+      name: 'Versus Todo List',
       tasks: {
-        count: 0,
-        todo: 0,
-        inProgress: 0,
-        list: []
+        count: 100,
+        todo: 40,
+        inProgress: 15,
+        complete: 45,
```

Comment:
Is there a reason you chose these values for initial values of the counts? They don't match the actual number of tasks in the various states. The first time I interact with the app (adding a task or clicking a button) the counts all readjust to what they should be. It seems you would want to start the counts so they matched the data if you are seeding with some data.